project/Build.scala

```diff
@@ -414,12 +414,6 @@ object DottyBuild extends Build {
         "org.scala-sbt" % "api" % sbtVersion.value % "test",
         "org.specs2" %% "specs2" % "2.3.11" % "test"
       ),
-      version := {
-        if (isNightly)
-          "0.1.1-" + VersionUtil.commitDate + "-" + VersionUtil.gitHash + "-NIGHTLY"
-        else
-          "0.1.1-SNAPSHOT"
-      },
```

Comment:
Previously this was done differently in `dotty-sbt-bridge` because of its different versioning scheme, now we can use the same `version` setting as `dotty` for `dotty-sbt-bridge`